evalml/tests/conftest.py

```diff
@@ -192,6 +192,15 @@ def ts_data():
     return X, y
 
 
+@pytest.fixture
+def ts_data_seasonal():
+    sine_ = np.linspace(-np.pi * 5, np.pi * 5, 500)
+    X, y = pd.DataFrame({"features": range(500)}), pd.Series(sine_)
+    y.index = pd.date_range(start='1/1/2018', periods=500)
+    X.index = pd.date_range(start='1/1/2018', periods=500)
```

Comment:
Nice. Does this `periods` arg mean we're creating 500 days starting on Jan 1 2018?

Reply:
Yes it creates 500 periods from Jan 1, 2018 with the default frequency of `daily`